test/unittest/statistics/dds/StatisticsQosTests.cpp

```diff
@@ -453,89 +445,85 @@ TEST_F(StatisticsFromXMLProfileTests, XMLConfigurationForStatisticsDataWritersQo
-                        </qos>                                                                                          \
-                </data_writer>                                                                                          \
-                <data_writer profile_name=\"_fastdds_statistics_discovered_entity\">                                    \
-                        <qos>                                                                                           \
-                                <reliability>                                                                           \
-                                        <kind>RELIABLE</kind>                                                           \
-                                </reliability>                                                                          \
-                        </qos>                                                                                          \
-                </data_writer>                                                                                          \
-        </profiles>                                                                                                     \
-        </dds>                                                                                                          \
-    ";
-
-    tinyxml2::XMLDocument xml_doc;
-    xml_doc.Parse(xml);
-    xml_doc.SaveFile("FASTRTPS_STATISTICS_PROFILES.xml");
+    const std::string xml =
+            "                                                                                                                 \
+            <?xml version=\"1.0\" encoding=\"utf-8\"  ?>                                                                      \
+                <dds xmlns=\"http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles\">                                          \
+                    <profiles>                                                                                                \
+                        <participant profile_name=\"statistics_participant\" is_default_profile=\"true\">                     \
+                            <rtps>                                                                                            \
+                                <propertiesPolicy>                                                                            \
+                                    <properties>                                                                              \
+                                        <property>                                                                            \
+                                            <name>fastdds.statistics</name>                                                   \
+                                            <value>HISTORY_LATENCY_TOPIC;PUBLICATION_THROUGHPUT_TOPIC;DISCOVERY_TOPIC;</value>\
```

Comment:
Instead of the alias it would be nice to have one check with the topic name directly, as it was done before: instead of `DISCOVERY_TOPIC`, use `_fastdds_statistics_discovered_entity`. This apply also in the corresponding profile.